From 4d356fb234f674abaec50be026804f0dcf1cc62d Mon Sep 17 00:00:00 2001
From: Daniel Borkmann <dborkman@redhat.com>
Date: Tue, 16 Sep 2014 15:04:40 -0400
Subject: [net] vxlan: remove unused port variable in vxlan_udp_encap_recv()

Message-id: <a71cbb774c0759f4daf6e0569f5958c6d39f8091.1410441647.git.dborkman@redhat.com>
Patchwork-id: 95086
O-Subject: [RHEL7.1 PATCH net 3/7] vxlan: remove unused port variable in vxlan_udp_encap_recv()
Bugzilla: 1130643
RH-Acked-by: David S. Miller <davem@redhat.com>
RH-Acked-by: Vlad Yasevich <vyasevic@redhat.com>
RH-Acked-by: Jiri Pirko <jpirko@redhat.com>

Bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=1130643
Upstream Status: 86c3f0f8307ac18f3ad3109e1969c62b8fbed5df
Brew: https://brewweb.devel.redhat.com/taskinfo?taskID=7947483
Tested: verified by IBM and myself

Upstream Commit:
commit 86c3f0f8307ac18f3ad3109e1969c62b8fbed5df
Author: Pablo Neira Ayuso <pablo@gnumonks.org>
Date:   Tue Feb 25 17:46:10 2014 +0100

    vxlan: remove unused port variable in vxlan_udp_encap_recv()

    Signed-off-by: Pablo Neira Ayuso <pablo@gnumonks.org>
    Signed-off-by: David S. Miller <davem@davemloft.net>

Signed-off-by: Daniel Borkmann <dborkman@redhat.com>
Signed-off-by: Jiri Benc <jbenc@redhat.com>
Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/net/vxlan.c b/drivers/net/vxlan.c
index c57fc24..7dd99cc 100644
--- a/drivers/net/vxlan.c
+++ b/drivers/net/vxlan.c
@@ -1118,7 +1118,6 @@ static int vxlan_udp_encap_recv(struct sock *sk, struct sk_buff *skb)
 {
  struct vxlan_sock *vs;
  struct vxlanhdr *vxh;
- __be16 port;
 
  /* Need Vxlan and inner Ethernet header to be present */
  if (!pskb_may_pull(skb, VXLAN_HLEN))
@@ -1136,8 +1135,6 @@ static int vxlan_udp_encap_recv(struct sock *sk, struct sk_buff *skb)
  if (iptunnel_pull_header(skb, VXLAN_HLEN, htons(ETH_P_TEB)))
   goto drop;
 
- port = inet_sk(sk)->inet_sport;
-
  vs = rcu_dereference_sk_user_data(sk);
  if (!vs)
   goto drop;
-- 
1.7.1